examples/aistore/aisio_usage_example.ipynb

```diff
@@ -0,0 +1,321 @@
+{
```

Comment:
@ejguan since Github doesnt offer good `ipynb` diffs do you think it makes sense to instead save the file as a regular python script using `jupyter nbconvert --to script 'my-notebook.ipynb'`?

Reply:
I still want to keep it as a notebook format as all the blocks can be rendered based on the type of code or comment. The ideal solution is probably to convert the notebook to a colab notebook that is runnable. WDYT?